### PR TITLE
add info for raspberry 4 with 4GB

### PR DESCRIPTION
**Pull request by bochachaner**
Since i had multiple reset with converting NEF file, i could increase the stability by changing the swap file and only use one worker.

### CODE REVIEW

File: docs/getting-started/troubleshooting/docker.md [hunk 1]
```diff
@@ -176,6 +176,29 @@ echo '/swapfile none swap sw 0 0' | tee -a /etc/fstab
...
+/swapfile file  100M  0B   -2
+```
+
+```bash
+sudo nano /etc/dphys-swapfile
+sudo reboot
+```
+Search for "CONF_SWAPSIZE=100" and increase the value at least to 4096. 
+
+Hint for raspberry 4 with 4GB 2048 is the maximul value which is used and to improve the sability set PHOTOPRISM_WORKERS to 1 in the docker-compose.yml.
```
Comment (alexislefebvre) [inline docs/getting-started/troubleshooting/docker.md:200]:
```suggestion
Hint for raspberry 4 with 4GB 2048 is the maximul value which is used and to improve the stability set `PHOTOPRISM_WORKERS` to 1 in the docker-compose.yml. 
```

**Comment by lastzero** [comment]
Thanks! Could you quickly fix the typos at the end?

**Comment by bochachaner** [comment]
done

**Comment by lastzero** [comment]
Improved the wording a little :)

**Pull request merged**